src/sgame/sg_extern.h

```diff
@@ -172,6 +170,9 @@ extern  vmCvar_t g_instantBuilding;
 
 extern  Cvar::Cvar<float> g_evolveAroundHumans;
 extern  Cvar::Cvar<float> g_devolveMaxBaseDistance;
+extern  Cvar::Cvar<float> g_friendlyFireAlienMultiplier;
```

Comment:
Why not put these in HealthComponent.cpp so you don't need to forward-declare them?